CITATION.cff

```diff
@@ -31,4 +31,4 @@ keywords:
   - pytorch
   - transformers
 license: Apache-2.0
-version: 0.11.1
+version: 0.12
```

Comment:
Usually, user pip install the package, so they get the latest version, not the dev